src/compiler/abstract_compiler.nit

```diff
@@ -814,6 +815,7 @@ extern void nitni_global_ref_decr( struct nitni_ref *ref );
 		v.add("signal(SIGTERM, sig_handler);")
 		v.add("signal(SIGSEGV, sig_handler);")
 		v.add("signal(SIGPIPE, sig_handler);")
+		v.add("setlocale(LC_ALL,\"\");")
```

Comment:
My issue is that on platform without gettext, this call and the
~~~
#include <locale.h>
~~~
will fail.

Either inject them only is some module are internationalized, or inject them trough the `gettext` module, for instance by refining run of Sys.

~~~nit
redef class Sys
   redef fun run do
      set_locale
      super
   end
   fun set_locale `{ setlocale(LC_ALL, ""); `}
end
~~~